Resources/Prototypes/Catalog/Fills/Crates/service.yml

```diff
@@ -18,6 +18,8 @@
         amount: 2
       - id: TrashBag
         amount: 2
+      - id: Plunger
```

Comment:
you may want to tweak the crate cargo order price if after adding all the prices of the items listed now, the actual value of the crate is higher than the sell price in the console